src/components/PledgeList.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import SinglePledge from './SinglePledge.jsx';
+
+const PledgeList = (props) => {
+  const { pledges } = props;
```

Comment:
I'm curious - for my own understanding - how is the assignment on **line 5**  different from the assignment in **src/components/App.jsx** on **line 41**

Is this redundant or required to get the pledges passed into the PledgeList's props?

Reply:
On line 41 in App.jsx when I pass "pledges" into the PledgeList component, I'm passing that data in as props, so they appear in PledgeList.jsx inside the props object. 
The props object then looks something like this: 
props = {
    pledges: [....]
}
So inside PledgeList, I do a spread operator to extract out the variable I want from props. One thing to note is that doing a spread operator becomes a lot more useful when there are a bunch of props being sent down into a child component, where here we only have one being passed down.

I could have omitted line 5 here, and then on line 8, I could have said props.pledges.map(...etc.. ). It's kind of a style thing that helps me organize.

Reply:
Okay great. This makes sense - I see how it's very useful in SinglePledge component where you are managing 7 different props!

Thanks for the explanation @christopher-tucker